source/xlsx/dffml_source_xlsx/xlsx.py

```diff
@@ -0,0 +1,66 @@
+from openpyxl import Workbook, load_workbook
+
+from dffml import config
+from dffml.util.entrypoint import entrypoint
+from dffml.source.file import FileSource
+from dffml.source.memory import MemorySource
+from dffml.record import Record
+
+
+@config
+class XLSXSourceConfig:
+    filename: str
+    readwrite: bool = False
+    allowempty: bool = False
+
+
+@entrypoint("xlsx")
+class XLSXSource(FileSource, MemorySource):
+    """
+    Source to read files in .xlsx format.
+    """
+
+    CONFIG = XLSXSourceConfig
+
+    async def load_fd(self, fd):
+        excel_file = load_workbook(fd.name)  # Loading the Workbook
```

Comment:
@pdxjohnny added the functionality for `load_fd`

Reply:
Please suggest if some additional modifications are required

Reply:
Let's add a `if not hasattr(fd.name): raise NotImplementedError(...)` in addition to creating the issue and the `# TODO` comment

Reply:
Yes we can have it , updating 👍